manifests/preinstall.pp

```diff
@@ -3,41 +3,22 @@
 # @api private
 class redis::preinstall {
   if $redis::manage_repo {
-    case $::operatingsystem {
-      'RedHat', 'CentOS', 'Scientific', 'OEL', 'Amazon': {
+    if $facts['os']['family'] == 'RedHat' {
```

Comment:
I'd prefer that the `case` statement is kept, so that it can have a `default` clause that fails instead of silently ignoring it.

Reply:
Hmm, I'll split it up in a different PR.